spec/models/work_package/work_package_planning_spec.rb

```diff
@@ -170,36 +170,25 @@
         expect(planning_element).not_to be_valid
 
         expect(planning_element.errors[:project]).to be_present
-        expect(planning_element.errors[:project]).to eq(["can't be blank"])
+        expect(planning_element.errors[:project]).to eq(["can't be blank."])
       end
     end
 
     describe 'parent' do
-      let (:de_message) { 'darf kein Meilenstein sein' }
```

Comment:
Was there a decision regarding the general location for localized specs? I'd argue this shouldn't be a concern here anymore since your refactoring is no longer locale-dependent.